controllers/topology/clusterclass_controller_test.go

```diff
@@ -265,11 +265,3 @@ func isOwnerReferenceEqual(a, b metav1.OwnerReference) bool {
 	return true
 }
 
-func ownerRefereceTo(obj client.Object) metav1.OwnerReference {
```

Comment:
Turns out we had two of those :)

Reply:
I remember this. I think it was because the 2 PRs that were adding this functions were in pipeline at the same time. :)